README_CN.md

```diff
@@ -63,6 +64,17 @@ Linkis 自2019年开源发布以来，已累计积累了700多家试验企业和
 请前往[Linkis releases 页面](https://github.com/apache/incubator-linkis/releases) 下载Linkis 的已编译版本或源码包。
 
 # 编译和安装部署
+```shell
+
+## 后端编译
```

Comment:
Whether to differentiate the OS?
For Win cmd  should be used as 
```
mvnw.cmd -N install
mvnw.cmd clean install -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
```

Reply:
ok, thanks.